src/components/Stepper.jsx

```diff
@@ -0,0 +1,59 @@
+import { React, useState } from "react";
+
+import Button from "./ui/Button";
+
+function Stepper(props) {
+    const { steps, currentStep } = props;
+    const [current, setCurrent] = useState(currentStep);
+    const nextStep = () => {
+        if (current < steps.length - 1) {
+            setCurrent(current + 1);
+        }
+    };
+
+    const prevStep = () => {
+        if (current > 0) {
+            setCurrent(current - 1);
+        }
+    };
+
+    return (
+        <div className='flex flex-col items-center justify-center'>
+            <div className='max-w-sm p-6 border-gray rounded-lg shadow dark:bg-gray-800 dark:border-gray-700'>
```

Comment:
You need to refer to our theme (tailwind.config.js)

use onlu colors in our theme, you can replace bg-gray-800 to bg-gray, since our gray color in our theme is a specific #value, and we dont provide palette.
Remove unused dark:bg-gray-800 dark:border-gray-700, since we do not intend to make a dark/light mode in the short term.

Reply:
PR is still Draft which means it is still under progress in local env.